wagtail/contrib/simple_translation/views.py

```diff
@@ -0,0 +1,153 @@
+from django.contrib import messages
+from django.contrib.admin.utils import unquote
+from django.core.exceptions import PermissionDenied
+from django.db import transaction
+from django.http import Http404
+from django.shortcuts import get_object_or_404, redirect
+from django.urls import reverse
+from django.utils.translation import gettext as _
+from django.utils.translation import gettext_lazy
+from django.views.generic import TemplateView
+from django.views.generic.detail import SingleObjectMixin
+from wagtail.admin.views.pages.utils import get_valid_next_url_from_request
+from wagtail.contrib.simple_translation.forms import SubmitTranslationForm
+from wagtail.core.models import Locale, Page, TranslatableMixin
+from wagtail.snippets.views.snippets import get_snippet_model_from_url_params
+
+
+class SubmitTranslationView(SingleObjectMixin, TemplateView):
```

Comment:
Perhaps rather than calling it "Submit translation", which I think implies there's some more complex translation management going on, could we rename this to something "Create translated page"/"Copy for translation"?